### PR TITLE
WS-1294: update banner for rebrand

### PR DESCRIPTION
**Pull request by srwang**
Storybook's responsive display looks different from the prod site's banner (it just shrinks the text size of everything to unreadable) so I wasn't really able to QA this, but it should be a quick fix forward if something's wrong. 

Screen Shot 2020-12-02 at 1 53 42 PM

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -45,11 +45,11 @@ const getPolicyHTML = (tag) => {
 
   if (tag === SPANISH_IETF_TAG) {
     const linkOpen = '<a href="https://edx.org/es/edx-privacy-policy" class="policy-link">';
-    return `edX y sus Miembros usan cookies y otras tecnologías de seguimiento para fines de rendimiento, análisis y marketing. Al usar este sitio web, aceptas este uso. Obtén más información sobre estas tecnologías en la ${linkOpen}Política de privacidad${linkClose}.`;
+    return `EdX y sus Miembros usan cookies y otras tecnologías de seguimiento para fines de rendimiento, análisis y marketing. Al usar este sitio web, aceptas este uso. Obtén más información sobre estas tecnologías en la ${linkOpen}Política de privacidad${linkClose}.`;
```
Comment (rgraber) [inline src/constants.js:48]: wait really? I thought we are still keeping everything lowercase

**Review by rgraber — COMMENTED** [review]
question about capitalization

File: src/constants.js (hunk 1 above)
Comment (AlasdairSwan) [reply src/constants.js:48]: That was my understanding too. In fact I just changed these all to lowercase ```e```s the other week.

**Review by AlasdairSwan — CHANGES_REQUESTED** [review]
I think it should be ```edX``` rather than ```EdX```. The rest looks good.

File: src/constants.js (hunk 1 above)
Comment (srwang) [reply src/constants.js]: it was EdX on the figma and my brain just got confused (forgot we're all lowercase)

**Comment by srwang** [comment]
Babel's been @babel/etc for a while so it may be that by grabbing the old repo path renovate isn't actually updating babel for us. The older version was causing problems when I tried to get the banner up using Storybook, although it seems to have worked just fine on prod.

**Review by rgraber — APPROVED** [review]
LGTM

**Review by AlasdairSwan — APPROVED** [review]
Make sure to update your commit message in line with semantic releases expectations :shipit:

**Review by albemarle — APPROVED** [review]

**Pull request merged**